### PR TITLE
Update m2install.sh

### PR DESCRIPTION
**Pull request by tshabatyn**

### CODE REVIEW

File: m2install.sh [hunk 1]
```diff
@@ -419,6 +419,12 @@ function createNewDB()
     mysqlQuery
 }
 
+function tuneAdminSessionLifetime()
+{
+    SQLQUERY="DELETE FROM ${DB_NAME}.core_config_data WHERE path = 'admin/security/session_lifetime'; INSERT IGNORE INTO ${DB_NAME}.core_config_data (scope, scope_id, path, value) VALUES ('default', 0, 'admin/security/session_lifetime', '31536000');";
```
Comment (yvoronoy) [inline m2install.sh]: Please add table prefix ${DB_NAME}.${TBL_PREFIX}.core_config_data

Comment (yvoronoy) [inline m2install.sh]: Instead two queries, please use insert on duplicate query.

File: m2install.sh [hunk 2]
```diff
@@ -451,7 +457,7 @@ function configure_files()
 {
     updateMagentoEnvFile
     overwriteOriginalFiles
-    CMD="find . -type d -exec chmod 775 {} \; && find . -type f -exec chmod 664 {} \;"
+    CMD="find . -type d -exec chmod 775 {} \; && find . -type f -exec chmod 664 {}"
```
Comment (yvoronoy) [inline m2install.sh]: This expression is invalid. Should be {} \; at the end.

File: m2install.sh [hunk 3]
```diff
@@ -976,9 +991,9 @@ function setProductionMode()
 
 function setFilesystemPermission()
 {
-    CMD="chmod u+x bin/magento"
+    CMD="chmod u+x ./bin/magento"
     runCommand
-    CMD="chmod -R 2777 ./var ./pub/media ./pub/static ./app/etc"
+    CMD="chmod -R a+rwX ./var ./pub/media ./pub/static ./app/etc"
```
Comment (yvoronoy) [inline m2install.sh]: a+rwX is equivalent to 0777 should be 2777
Consider to leave it as is 2777

**Comment by tshabatyn** [comment]
CR fixed

**Comment by yvoronoy** [comment]
@tshabatyn Thank you for your contribution 👍

**Pull request merged**